app/assets/stylesheets/course/course_users.scss

```diff
@@ -0,0 +1,8 @@
+.course-course-users {
+  .profile-picture {
+    .image > img {
+      margin-top: 1em;
+      max-height: $course-course-user-profile-picture;
```

Comment:
I guess the look will be more consistent if the size of the image does not change. Will use `height` and let the wording go below the pic on small screens.